TESTING.md

```diff
@@ -328,3 +321,73 @@ imported packages: **o1, o1.llama, o1.randomtext**.
 **8.4** Double click **SwingExamples** in **Modules** list.
 
 **8.5** ASSERTION: **SwingExamples** shows up as a module in the project tree.
+
+### 9 Authentication with A+ LMS
+
+**9.1** From the main menu, choose **A+ > Set A+ Token**.
+
+![Set token](images/setToken.png)
+
+**9.2** Click on the highlighted "What's my token?" link.
+
+**9.3** ASSERTION: Browser opens your profile page in A+ LMS (login, if asked).
+
+**9.4** Copy **API Access Token** from the web page and insert it into the prompt, click "OK".
+
+**9.5** ASSERTION: The assignments tab is populated with Weekly assignments.
```

Comment:
Nit: "weekly" unnecessarily capitalized? Perhaps
"ASSERTION: The assignments tab lists weeks of the course."